src/chips/nrf51822/rtc.rs

```diff
@@ -0,0 +1,139 @@
+use core::mem;
```

Comment:
Do we thinks this works now? If so I'll review it, but otherwise maybe we can take the RTC out of this pull request and try to merge it separately?